README.md

```diff
@@ -80,12 +80,13 @@ This repo contains subgraphs defined using a set of standardized schemas. These
-| [SpookySwap](https://spooky.fi/) | 🛠 | 1.3.0 / 1.1.3 / 1.0.0 | [![SpookySwap Fantom](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/spookyswap-fantom) |
-| [SushiSwap](https://www.sushi.com/) | 🛠 | 1.3.0 / 1.1.5 / 1.0.0 | [![SushiSwap Ethereum](./docs/images/chains/ethereum.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-ethereum) [![SushiSwap Celo](./docs/images/chains/celo.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-celo) [![SushiSwap Fuse](./docs/images/chains/fuse.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-fuse) [![SushiSwap Moonriver](./docs/images/chains/moonriver.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-moonriver) [![SushiSwap Moonbeam](./docs/images/chains/moonbeam.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-moonbeam) [![SushiSwap Avalanche](./docs/images/chains/avalanche.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-avalanche) [![SushiSwap BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-bsc) [![SushiSwap Arbitrum](./docs/images/chains/arbitrum.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-arbitrum) [![SushiSwap Fantom](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-fantom) [![SushiSwap Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-polygon) [![SushiSwap Gnosis](./docs/images/chains/xdai.png)](https://thegraph.com/hosted-service/subgraph/messari/sushiswap-gnosis) |
-| [Trader Joe](https://traderjoexyz.com/home#/) | 🛠 | 1.3.0 / 1.0.3 / 1.0.0 | [![Trader Joe Avalanche](./docs/images/chains/avalanche.png)](https://thegraph.com/hosted-service/subgraph/messari/trader-joe-avalanche) |
-| [Ubeswap](https://ubeswap.org/) | 🛠 | 1.3.0 / 1.1.4 / 1.0.0 | [![Ubeswap Celo](./docs/images/chains/celo.png)](https://thegraph.com/hosted-service/subgraph/messari/ubeswap-celo) |
+| [Solarbeam](https://solarbeam.io/) | 🛠 | 1.3.0 / 1.1.3 / 1.0.0 | [![Solarbeam Moonriver](./docs/images/chains/moonriver.png)](https://thegraph.com/hosted-service/subgraph/messari/solarbeam-moonriver) |
+| [SpiritSwap](https://www.spiritswap.finance/) | 🛠 | 1.3.0 / 1.0.2 / 1.0.0 | [![SpiritSwap Fantom](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/spiritswap-fantom) |
+| [SpookySwap](https://spooky.fi/) | 🛠 | 1.3.0 / 1.1.4 / 1.0.0 | [![SpookySwap Fantom](./docs/images/chains/fantom.png)](https://thegraph.com/hosted-service/subgraph/messari/spookyswap-fantom) |
+| [Ubeswap](https://ubeswap.org/) | 🛠 | 1.3.0 / 1.1.5 / 1.0.0 | [![Ubeswap Celo](./docs/images/chains/celo.png)](https://thegraph.com/hosted-service/subgraph/messari/ubeswap-celo) |
```

Comment:
Note that I moved things around a little bit today because they weren't alphabetical. You should keep these changes so they are sorted: https://github.com/messari/subgraphs/commit/9da410ae20f4036ab07c2e48d1917cc91b591008

Reply:
Alright! My bad. I fixed this.